### PR TITLE
I've added a note about automation for clusters/services

### PR DESCRIPTION
**Pull request by ay-b**
## Pull Request Checklist

**Is this in reference to an existing issue?**
No, it's a new external feature.

#### Purpose

Automate the process of creation of multiple jsons, when there are a lot of services have to be monitored. Added a link to my script to the NOTES section of the plugin.

#### Known Compatibility Issues

None found yet

### CODE REVIEW

File: bin/check-ecs-service-health.rb [hunk 1]
```diff
@@ -27,6 +27,13 @@
 #
 # NOTES:
 #
+<<<<<<< HEAD
```
Comment (majormoses) [inline bin/check-ecs-service-health.rb]: remnant of a bad merge

File: bin/check-ecs-service-health.rb [hunk 2]
```diff
@@ -27,6 +27,13 @@
...
+=======
```
Comment (majormoses) [inline bin/check-ecs-service-health.rb]: remnant of a bad merge

File: bin/check-ecs-service-health.rb [hunk 3]
```diff
@@ -27,6 +27,13 @@
...
+# MULTIPLE CLUSTERS/SERVICES AUTOMATION
+#   Create multiple clusters/services with these scripts: https://github.com/ay-b/ecs-service-check-autocreate
+#   Don't forget to edit template file
+#
+>>>>>>> check-ecs-service-health
```
Comment (majormoses) [inline bin/check-ecs-service-health.rb]: remnant of a bad merge

**Comment by ay-b** [comment]
I'm sorry, I don't get about what lines you are talking. 
I'm relatively new to git-process, so, please, forgive me in forward.

**Comment by majormoses** [comment]
I would do some general reading on git merging and rebasing as it will help you in cases where it is more complex than this. essentially in this case I believe you can just kill the lines that I commented on. Normally when doing a merge you select one of the two segments of code you wish to keep.

**Comment by mbbroberg** [comment]
Hey @ay-b, you ran into a merge conflict[1], which results in lines like the ones @majormoses pointed out. Please delete all of the lines he commented on (30, 31, 36) and your PR can then be merged. And thank you! ❤️ 

[1] Read more on merge conflicts and resolving them here: https://githowto.com/resolving_conflicts

You can also quickly see these issues if you use a text editor with merge conflict resolution. I love how [Atom](https://github.atom.io/) handles this. Click `merge conflict` on this page and see what I mean 😄

**Comment by majormoses** [comment]
@ay-b I rebased and fixed the merge conflicts for you. Can you spot check it to make sure I did not goof anything before I merge this?

**Pull request merged**